packages/react-fluentui-examples/List/list-active-element/src/main.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import { FluentProvider, webLightTheme } from '@fluentui/react-components';
+
+import Example from './List-Active-Element.tsx';
+
+ReactDOM.createRoot(document.getElementById('root')!).render(
+  <React.StrictMode>
+    <FluentProvider theme={webLightTheme}>
+      <Example />
```

Comment:
Let's add some kind of mounting/routing support - Otherwise there's too much barrier for someone to contribute the 2nd,3rd examples because then they would need to build that